osu.Game/Database/LegacyExporter.cs

```diff
@@ -19,6 +20,16 @@ namespace osu.Game.Database
     public abstract class LegacyExporter<TModel>
         where TModel : class, IHasNamedFiles
     {
+        /// <summary>
+        /// Max length of filename (including extension).
+        /// </summary>
+        /// <remarks>
+        /// TODO: WHY? DOCUMENTATION PER PLATFORM LINKS HERE.
```

Comment:
Again, needs documentation here. Including why we want to apply this to all platforms (just for simplicity?) and which platforms feature this limit. Including links to documentation for each platform's documentation on the issue.

Reply:
> just for simplicity?

Technically yes, max file name depends on [fs limits](https://en.wikipedia.org/wiki/Comparison_of_file_systems#Limits) and system configuration (like [MAX_PATH](https://learn.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=registry)). It would be really hard to get for every system.